usr/share/rear/prep/GNU/Linux/220_include_lvm_tools.sh

```diff
@@ -1,10 +1,12 @@
-# Include LVM tools if LVM exists
+# Always include DM tools
```

Comment:
please add an explanation why dmsetup and dmeventd are always included
and also add the URL of your pull request as a comment, cf. the section
"Code must be easy to understand (answer the WHY)" in
https://github.com/rear/rear/wiki/Coding-Style